resources-generator/src/main/kotlin/dev/icerock/gradle/generator/android/AndroidMRGenerator.kt

```diff
@@ -21,6 +21,17 @@ class AndroidMRGenerator(
     mrClassPackage = mrClassPackage,
     generators = generators
 ) {
+
+    init {
+        assetsGenerationDir.mkdirs()
+        sourceSet.addAssetsDir(assetsGenerationDir)
+    }
+
+    override fun beforeMRGeneration() {
+        super.beforeMRGeneration()
+        assetsGenerationDir.deleteRecursively()
```

Comment:
this should be in base generator, not android-only

Reply:
All targets except android don't have assets path. (Not sure about jvm). So that's why operations with the folder only happen on android.

Reply:
i think we should have all 3 types in base generator - sources, resources, assets. and each platform can decide where store this types - only android will move it to separated dir, all others will use directory inside resources. in this case logic of generation will cleaner